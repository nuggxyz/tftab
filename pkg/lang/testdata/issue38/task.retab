

gen file_a {
	path = "../some/nested/dir/tmp.yaml"
	data = {
		chicken = "bock bock"
	}
}

gen file_b {
	path = "../some/nested/new.yaml"
	data = {
		goat = gen.file_a.resolved_output
		cow = "gwaitnaadfasdfsdsdfsa what ? ${gen.file_a.resolved_output}"
	}
}
