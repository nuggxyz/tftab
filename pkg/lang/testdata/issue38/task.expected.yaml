# code generated by retab. DO NOT EDIT.
# join the fight against yaml @ github.com/walteh/retab

# source: "task.retab"

goat: dir/tmp.yaml
cow: gwaitnaadfasdfsdsdfsa what ? dir/tmp.yaml
